tests/phpunit/unit-tests/admin/post-types/meta-boxes/class-llms-test-meta-box-order-details.php

```diff
@@ -227,4 +269,140 @@ public function test_save_remaining_payments_success() {
 
 	}
 
+	/**
+	 * Test save_remaining_payments() when changes are made but the order doesn't support recurring payment modifications.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_save_remaining_payments_unsucces_when_order_does_not_support_recurring_payment_modifications() {
```

Comment:
```suggestion
	public function test_save_remaining_payments_unsuccess_when_order_does_not_support_recurring_payment_modifications() {
```
This method name reminds me of the South Park movie title "Bigger, Longer & Uncut".